webapp/Component.js

```diff
@@ -1,7 +1,11 @@
-sap.ui.define(["sap/ui/core/UIComponent", "sap/ui/core/ComponentSupport"], (UIComponent) => {
+sap.ui.define(["sap/ui/core/UIComponent", 
+	"sap/ui/core/ComponentSupport"
+], (UIComponent, ComponentSupport) => {
 	"use strict";
+
 	return UIComponent.extend("sap.ui.demo.todo.Component", {
 		metadata: {
+			// interfaces: ["sap.ui.core.IAsyncContentCreation"],
```

Comment:
why was this added?

Reply:
![image](https://github.com/HansenLudwig/openui5-sample-app/assets/60106776/dfeb1cc3-6f19-4536-8c70-826cc44a45f9)
Hi, we can check if it can be seen after this Pull Req..
As shown in the Pic, I deleted this comment and added a new comment here.
 - VG